compiler/bindings/c/iree/compiler/dialects/iree_gpu.h

```diff
@@ -93,6 +93,23 @@ MLIR_CAPI_EXPORTED MlirAttribute ireeGPULoweringConfigAttrGet(
 MLIR_CAPI_EXPORTED MlirAttribute
 ireeGPULoweringConfigAttrGetAttributes(MlirAttribute attr);
 
+MLIR_CAPI_EXPORTED void
+ireeGPULoweringConfigAttrGetReductionTileSizes(MlirAttribute attr, size_t *len,
+                                               int64_t *reductionTileSizes);
+
+MLIR_CAPI_EXPORTED void
+ireeGPULoweringConfigAttrGetWorkgroupTileSizes(MlirAttribute attr, size_t *len,
+                                               int64_t *workgroupTileSizes);
```

Comment:
I'd put these in a single function that returns all tile sizes in a struct. You can see an example above (`ireeGPUMMAAttrGetInfo`). The reason is that this is expands the API surface area by quite a lot, and adding more levels of tiling would exacerbate this further. With a struct, we can keep extending it with more fields.